surrogates/common_utils/fits.py

```diff
@@ -0,0 +1,139 @@
+##==============================================================================
+## BHPTNRSurrogate module
+## Description : evaluates the (spline or GPR) surrogate model
+## Author : Tousif Islam, Nov 2022 [tislam@umassd.edu / tousifislam24@gmail.com]
+##==============================================================================
+
+import numpy as np
+import scipy
+from scipy.interpolate import splrep, splev
+from . import utils
+from .eval_pysur import evaluate_fit as evaluate_GPR
+
+#----------------------------------------------------------------------------------------------------
+def evaluate_GPR_at_EIM_nodes(X, fit_data):
+    """ Evaluate the spline at one EIM node """
+    return None
+
+#----------------------------------------------------------------------------------------------------
+def evaluate_splines_at_EIM_nodes(X, fit_data):
+    """ Evaluate the spline at one EIM node """
+    
+    [h_eim_spline, eim_indicies] = fit_data
+    return np.array([splev(X, h_eim_spline[j]) for j in range(len(eim_indicies))])
+
+
+#----------------------------------------------------------------------------------------------------
+def EIM_B_to__waveform_datapiece(B, eim_vals):
+    """ Compute the interpolated waveform for a single mode : dominant 22 mode """
```

Comment:
does this routine only work for the 22 mode? seems more generic than that -- looks like any data piece (amp, phase, etc). maybe reword the documentation to mention its for generic data pieces (e.g. the 22 mode's amplitude and phase)